master/internal/api_workspace_intg_test.go

```diff
@@ -521,3 +522,29 @@ func TestAuthzWorkspaceGetThenActionRoutes(t *testing.T) {
 		require.Equal(t, expectedErr.Error(), curCase.IDToReqCall(id).Error())
 	}
 }
+
+func TestWorkspaceHasModels(t *testing.T) {
+	// set up a dB and api server to use for integration testing
+	require.NoError(t, etc.SetRootPath("../static/srv"))
+	pgDB, cleanup := db.MustResolveNewPostgresDatabase(t)
+	defer cleanup()
+	db.MustMigrateTestPostgres(t, pgDB, "file://../static/migrations")
```

Comment:
```suggestion
```

These lines setup a fresh postgres database. It is more expensive from a test execution time perspective and also makes it harder to debug so we shouldn't do this when we don't need to. It would be great if we avoid doing this unless there is a good reason.

GetWorkspaces does this because it is hard to test listing workspaces when other tests can add or there can be preexisting workspaces.